devops/azure-pipelines/code-coverage-analysis-template.yml

```diff
@@ -0,0 +1,30 @@
+parameters:
+  - name: 'serviceConnection'
+    type: string
+  - name: 'organization'
+    type: string
+  - name: 'projectKey'
+    type: string
+  - name: 'projectName'
+    type: string
+jobs:
+  - job: Static_Analysis
+    steps:
+      - task: SonarCloudPrepare@1
+        inputs:
+          SonarCloud: '${{ parameters.serviceConnection }}'
+          organization: '${{ parameters.organization }}'
+          scannerMode: 'CLI'
+          configMode: 'manual'
+          cliProjectKey: '${{ parameters.projectKey }}'
+          cliProjectName: '${{ parameters.projectName }}'
+          cliSources: './src'
+        displayName: Sonar Configuration
```

Comment:
we'd really like to have had code coverage by this point, it's pretty late in the iteration already. i guess it doesn't need to block this PR but it should also not break the devs' workflow, so the quality gate needs to not require any code coverage  until you're actually sending it to sonar.

Reply:
Quality Gate now ignores code coverage (in the proper way), and I'll talk with devs on Monday about it.